generating_an_expression_matrix/README.md

```diff
@@ -61,98 +61,120 @@ This is what the your history will look like:
 
 ## 2. The workflow
 
-Alevin collapses the steps involved in dealing with dscRNA-seq into a single process, so much of what we'll cover is 'plumbing' of inputs, and interpretation. 
+Alevin collapses the steps involved in dealing with dscRNA-seq into a single process, so much of what we'll cover is 'plumbing' of inputs, and interpretation.  
 
 ### 1. Generate a transcriptome index
 
-Normally, you would need to generate an index for your transcriptome of interest, containing all sequences likely to be present in your experiment's read data. Generating that index is too time-consuming to do as part of this sesssion, so we'll using using one we made earlier. But have a look at the fasta format sequences, which are the same as the ones we used to build the index. 
+A transcriptome index contains all of the transcript sequences and corresponding transcript names likely to be present in your experiment's read data. Normally, you would need to first generate the transcript sequence list and then align these to their corresponding transcript names or identifiers, thus generating the transcriptome index. Is there a general transcriptome index for each species? Of course! Without delving further, however, note that not all transcriptome indices of the same species are, in fact, the same! However, generating a custom transcriptome index is too time-consuming to do as part of this session, so we'll use one a built-in index. But do have a look at the fasta format sequences provided, which are the same as the ones we used to build the index.   
 
 ### 2. Generate a transcript to gene mapping
 
-To generate gene-level quantifications based on transcriptome quantification, Alevin and similar tools require a conversion between transcript and gene identifiers, which we can derive from the annotations available in genome resources such as Ensembl. The transcripts in this list need to match the ones we used in the sequences reference. The study we're working with here added spike-ins to their samples, so when we created the index above in preparation for this tutorial, we combined sequence information for the biological sequences and the spike-ins. So now we need to generate a transcript to gene mapping with both types of sequence.
+Gene-level, rather than transcript-level, quantification is standard in scRNA-seq, which means that that alternatively spliced RNA molecules count as the same gene. In a nutshell, this quantification is used because droplet-based scRNA-seq use short, single-ended reads, which can make it impossible to distinguish transcript isoforms.
```

Comment:
I know I'm being picky here, but I'm going to pick some more I'm afraid. Single-endendess is not the issue, length is not really the issue. It's the fact that, in contrast to smart-like tech, the reads come from one end of the molecule, rather than being scattered through it. Honestly, I think we should just keep things simple and say that Alevin works from a transcriptome input, but outputs results at the gene level, and therefore needs a mapping.

Reply:
Please do be picky, no apologies needed! It should be right. The issue is, even as a wet-labber I get confused at the need for gene vs transcript level quantification and since this is so early in the protocol, people are going to want to know why they are doing this. "The tool works like this" is not the answer here (though I recognise often IS the answer!).  What if I put: 
In a nutshell, this quantification is used because droplet-based scRNA-seq use short reads from one end of the transcript molecule,  which makes it impossible to consistently distinguish transcript isoforms.

Reply:
The thing is, some people do claim that you can do some differential abundance with droplet metrics, so I don't like making categorical statements like that. You probably could have a go in some limited circumstances with short exons, different start points etc, but Alevin doesn't offer that functionality. Pipelines are dictated by the tools available I'm afraid. The fact that differential isoform analysis is difficult with methods like this is sort of a separate issue.